src/components/OpenWith/OpenWith.js

```diff
@@ -292,20 +293,25 @@ class OpenWith extends PureComponent<Props, State> {
      * @return {Element}
      */
     render() {
-        const { language, messages: intlMessages }: Props = this.props;
+        const {
+            language,
+            messages: intlMessages,
+            className,
+        }: Props = this.props;
         const {
             fetchError: error,
             isLoading,
             integrations,
             executePostData,
         }: State = this.state;
 
+        const styleClassName = classNames('be bcow', className);
```

Comment:
May want to consider doing `const className = classNames('be bcow', this.props.className);`.

Reply:
sounds reasonable